### DISCUSSION TITLE
Dynamic CachedFont in flutter

### DISCUSSION THREAD

**Discussion by expoders8** [sidrao2006/dynamic_cached_fonts#214] [Q&A]
can we store multiple cached fonts in it ?? 
Also we can get cached font list from this package??

Thank You sir ...!!!

**Comment by sidrao2006** [comment]
Hello,

If you're trying to load multiple fonts that belong to a single font family (for example, the italic or bold version of a font along with the regular version), then you'll want to take a look at [`DynamicCachedFonts.family`](https://pub.dev/documentation/dynamic_cached_fonts/latest/dynamic_cached_fonts/DynamicCachedFonts/DynamicCachedFonts.family.html)

If you want to load multiple distinct fonts, then you could download them using [`cacheFont`](https://pub.dev/documentation/dynamic_cached_fonts/latest/dynamic_cached_fonts/DynamicCachedFonts/cacheFont.html) to download each one of the fonts individually and then make them available for use by calling [`loadCachedFont`](https://pub.dev/documentation/dynamic_cached_fonts/latest/dynamic_cached_fonts/DynamicCachedFonts/loadCachedFont.html) for each of the fonts.

Unfortunately, at the moment, listing cached fonts is not supported due to the limitations of the underlying caching mechanism.
That said, I'd be happy to look into it if you want.
